src/cloudai/_core/installables/file.py

```
@@ -0,0 +1,72 @@
+    def __eq__(self, other: object) -> bool:
+        return isinstance(other, File) and other.src == self.src
+
+    def __hash__(self) -> int:
+        return hash(self.src)
+
+    def install(self, installer: "BaseInstaller") -> InstallStatusResult:
+        self.installed_path = installer.system.install_path / self.src.name
+        shutil.copyfile(self.src, self.installed_path, follow_symlinks=False)
+        return InstallStatusResult(True)
+
+    def uninstall(self, installer: "BaseInstaller") -> InstallStatusResult:
```

Ruff (0.15.12)

[warning] Unused method argument: `installer`

(ARG002)